services/src/handlers/wfs.rs

```diff
@@ -189,6 +189,19 @@ async fn get_feature<C: Context>(
         .initialize(&execution_context)
         .context(error::Operator)?;
 
+    // handle request and workflow crs matching
+    let spatial_ref = initialized.result_descriptor().spatial_reference();
+    // TODO: use a default spatial reference if it is not set?
+    snafu::ensure!(request.srs_name.is_some(), error::InvalidSpatialReference);
```

Comment:
4326 by default?

Reply:
i really don't know. i guess the default is to use the projection of the dataset or in our case the workflow